snap/snapcraft.yaml

```diff
@@ -24,34 +16,24 @@ parts:
   celeste:
     plugin: dump
     source: .
-    source-type: git
+    parse-info: ["usr/share/metainfo/com.hunterwittenborn.Celeste.metainfo.xml"]
     build-packages:
       - golang-go
       - just
-      - libadwaita-1-dev
-      - libatk1.0-dev
-      - libcairo2-dev
       - libclang-15-dev
-      - libgdk-pixbuf-2.0-dev
-      - libglib2.0-dev
-      - libgraphene-1.0-dev
-      - libgtk-3-dev
-      - libgtk-4-dev
-      - libpango1.0-dev
-      - pkg-config
       - rustup
-    stage-packages:
-      - libadwaita-1-0
-      - libayatana-appindicator3-1
-      - libgtk-3-0
-      - rclone
     override-build: |
       just build
-    override-prime: |
-      craftctl default
-      cd ~/parts/celeste/build
-      DESTDIR=~/prime just install
-      sed -i 's|Icon=.*|Icon=${SNAP}/usr/share/icons/hicolor/scalable/apps/com.hunterwittenborn.Celeste.svg|' ~/prime/usr/share/applications/com.hunterwittenborn.Celeste.desktop
+      DESTDIR=$CRAFT_PART_INSTALL just install
+
+  deps:
+    after:
+      - celeste
+    plugin: nil
+    stage-packages:
+      - rclone
+    prime:
+      - usr/bin/rclone
```

Comment:
Is there a reason we're doing this all as a separate step instead of just doing it in the previous one?

Reply:
So, that later we can simply add other packages, without cluttering the main part.

Reply:
I feel like the `snapcraft.yaml` file is still quite legible even with it all in one section. Does it affect the resulting package in any meaningful way? If it doesn't I'd probably want to stick with it being all in one section.